app/build.gradle

```diff
@@ -51,6 +51,15 @@ dependencies {
       'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-alpha03',
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
   )
+  implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+  implementation 'com.google.android.material:material:1.0.0'
+  implementation 'androidx.appcompat:appcompat:1.0.2'
+  implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+  implementation 'androidx.navigation:navigation-fragment:2.0.0'
+  implementation 'androidx.navigation:navigation-ui:2.0.0'
+  implementation 'androidx.lifecycle:lifecycle-extensions:2.0.0'
+  implementation 'androidx.navigation:navigation-fragment-ktx:2.0.0'
+  implementation 'androidx.navigation:navigation-ui-ktx:2.0.0'
```

Comment:
Combine all these `implementation` together, also preferably in alphabetical order. 
Reference: https://github.com/oppia/oppia-android/blob/develop/domain/build.gradle

Reply:
okay